### PR TITLE
FIX: typo

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: metadatastore/mds.py [hunk 1]
```diff
@@ -23,7 +23,7 @@ def __init__(self, config, version=1):
         self.version = version
 
     def reset_caches(self):
-        self._RUNSTART_CAHCE.clear()
+        self._RUNSTART_CACHE.clear()
         self._RUNSTOP_CAHCE.clear()
```
Comment (klauer) [inline metadatastore/mds.py]: This and next line too

Comment (danielballan) [inline metadatastore/mds.py]: Bah, you _just_ beat me to it.

Comment (klauer) [inline metadatastore/mds.py]: 😄

**Comment by tacaswell** [comment]
`mds_all` includes the `commands` module as one of the categories,  I think there is a fixture which is just the two mds objects.

**Comment by danielballan** [comment]
bump @arkilic @tacaswell This works now that commands is gone.

**Pull request merged**